### DISCUSSION TITLE
Okulda alınan dersler sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#265] [Zeka Sorusu]
İki kız ve üç erkek öğrencinin kayıtlı olduğu bir okulda, öğrencilerin bu dönem aldıkları dersler ile ilgili aşağıdaki bilgiler verilmiştir.

 Matematik dersine çift sayıda öğrenci kayıtlıdır
 Fizik dersinde kız öğrencilerin sayısı erkek öğrencilere eşittir
 Ali ve Fatma Kimya dersine kayıtlı öğrencilerden ikisidir
 Ahmet ve Mehmet Matematik dersine kayıtlı öğrencilerden ikisidir
 Biyoloji dersini 3 öğrenci almaktadır
 Bir öğrenci bir dönemde tam olarak 3 derse kayıtlıdır
 Bir derste en az bir tane öğrenci kayıtlı olmalıdır
 Ali ve Mehmet hem Biyoloji hem de Türkçe derslerine kayıtlıdır
 Fatma Fizik dersine kayıtlı değildir
 Türkçe dersine hiçbir kız öğrenci kayıtlı değildir
Yukarıda verilen bilgiler ışığında aşağıdaki soruları yanıtlayınız.

Soru 1: Kimya dersini kaç öğrenci almaktadır?

Soru 2: Ayşe hangi dersleri almaktadır?

Soru 3: Fatma ve Ayşe’nin ortak aldığı dersler hangileridir?

Soru 4: Matematik dersini Ahmet ve Mehmet’e ek olarak kimler almaktadır?

Soru 5: Fizik dersini kimler almaktadır?

**Comment by Alperg123456** [comment]
Soru 1: Kimya dersini kaç öğrenci almaktadır?
Cevap 1: 3 kişi (Ali, Fatma, Ayşe)

Soru 2: Ayşe hangi dersleri almaktadır?
Cevap 2: Fizik, Matematik, Kimya

Soru 3: Fatma ve Ayşe’nin ortak aldığı dersler hangileridir?
Cevap 3: Matematik, Kimya

Soru 4: Matematik dersini Ahmet ve Mehmet’e ek olarak kimler almaktadır?
Cevap 4: Ayşe, Fatma

Soru 5: Fizik dersini kimler almaktadır?
Cevap 5: Ayşe, Ahmet
